### PR TITLE
Add command lib in golang workspace

### PR DESCRIPTION
**Pull request by Xenorith**
adds command/command.go to be used for future golang development

### CODE REVIEW

File: dev/scripts/src/alluxio.org/go.mod [hunk 1]
```diff
@@ -2,4 +2,7 @@ module alluxio.org
 
 go 1.12
```
Comment (jenoudet) [inline dev/scripts/src/alluxio.org/go.mod:3]: ?
```suggestion
go 1.18
```

File: dev/scripts/src/alluxio.org/command/command.go [hunk 2]
```diff
@@ -0,0 +1,224 @@
+/*
```
Comment (jenoudet) [inline dev/scripts/src/alluxio.org/command/command.go:1]: This is a copy of the one in the private repo. Will this be replaced by an import eventually?

**Review by jenoudet — CHANGES_REQUESTED** [review]

File: dev/scripts/src/alluxio.org/command/command.go (hunk 2 above)
Comment (Xenorith) [reply dev/scripts/src/alluxio.org/command/command.go:1]: not in the near future. if there becomes significant overlap, then we'd consider doing imports but the coupling is too weak at the moment to incur the overhead of establishing a dependency relationship.

**Comment by Xenorith** [comment]
a separate problem to eventually deal with: turn off java builds for PRs with only golang changes x.x

**Review by jenoudet — APPROVED** [review]

**Comment by Xenorith** [comment]
alluxio-bot, merge this please

**Pull request merged**